### ISSUE TITLE
[DigitalStrom] OpenHAB 2.5 digitalStrom Binding: crashes DigitalStrom-Server

### ISSUE THREAD

**Issue by oli-h** [openhab/openhab-addons#6679]
OpenHAB's digitalStrom-binding seems to stress the DSS ([DigitalStrom-Server dss20](https://productinfo.digitalstrom.com/7640156791747/)) so hard that it finally does no more answer on its REST-port 8080.

Also the 'own, internal' UI from the DigitalStrom-Server does not work any more. Connections to TCP-port 8080 are all 'pending' (seen in Chrome Dev-Tools)
Need to 'reset' the DigitalStrom-Server with a physical 'Button-Push' on the device itself.
This happens again and again, typically after a few hours of OpenHAB running. Finally need to disable OpenHAB (i.e.: simply not starting Docker-Container) - and then uninstalling OpenHAB (as it makes no sense to me when it does not interact properly with my digitalStrom-infrastructure)

## Expected Behavior
OpenHAB should not crash the DigitalStrom-Server but integrate with it.

## Possible Solution
no idea...

## Context
Read-Out every 30 seconds:
- total power consumtion (Watts) from DigitalStrom-Server
- individual power consumption of 7 (seven) digitalStromg-Meter

only one plain 'light switch' set-up in OpenHAB

## Your Environment
- Linux (Mint 19.2), Using OpenHAB's docker image `openhab/openhab:2.5` _as is_
- dss20 with seven digitalStrom-Meters, dss-Firmware up-to-date (i.e. 1.16.3)
- all in same LAN

**Comment by 9037568** [comment]
Need your configuration and a debug log, please.

**Comment by hmerk** [comment]
There seem to be issues in combination with newer dss servers and firmware. I am running an old DSS11 server with latest firmwaren (1.14.5) and openhab never crashed the dss since roughly about 6 years now.

**Comment by oli-h** [comment]
Tried to enable Log-Level - but had no luck. I've just seen Jetty's http-Logs (Header-parsing etc.).
There seem to be no logs from OpenHAB's "digitalStrom"-Stuff.

**Comment by 9037568** [comment]
https://twitter.com/oltuck27/status/630384054295265281

Get thee to the community forum and read the "Taming openHAB logging" thread.

**Comment by oli-h** [comment]
Sorry, guys, meanwhile I detected a completly different cause of my described problems:
Ethernet-network-link between my switch any my digitalSTROM-server seems to be shaky.

Though, I use this setup now for > 2 years without problems it seems that the "OpenHAB queries the dss-Server **regulary** via its REST-API-on-Port-8080" now discovered the problem.
Typically, this network-link idles as nearly all is done "digitalSTROM-internally" - only firmware-updates and a few settings in the very first days required an intensive use of digitalSTROM's WebInterface. Therefore, the shaky network-links was unknown.

I now reduced to "10 MBit Ethernet" (yeah, really - still far enough for a 'command to switch on a light') and with this it looks stable. Will replace the ethernet-cable soon and try again with "full" 100-MBit (max. for digitalSTROM-Server)
